src/ecmascript_simd.js

```diff
@@ -170,11 +176,7 @@ function simdFrom(toType, fromType, a) {
 
 function simdFromBits(toType, fromType, a) {
   a = fromType.fn.check(a);
-  for (var i = 0; i < fromType.lanes; i++)
-    fromType.buffer[i] = fromType.fn.extractLane(a, i);
-  for (var i = 0; i < toType.lanes; i++)
-    lanes[i] = toType.buffer[i];
-  return simdCreate(toType);
+  return simdCreateFromBuffer(toType, a.s_.buffer);
```

Comment:
Could you just directly invoke the constructor function to get a new instance without 'lanes'? Also, I think it's slightly clearer if you just inline simdCreateFromBuffer.

function simdFromBits(toType, fromType, a) {
   a = fromType.fn.check(a);
  var newValue = new toType.fn();
  newValue.s_ = new type.view(a.s_.buffer);
  return newValue;
}